xdp-dump/tests/test-xdpdump.sh

```diff
@@ -65,6 +65,8 @@ END
 
 test_interfaces()
 {
+    assert_kernel_version_greater_or_equal 5.7.0
```

Comment:
Don't think you should assert here, as only the output is different it can still be tested for correctness.